ovs/lib/storagedriver.py

```diff
@@ -169,7 +167,7 @@ def scheduled_voldrv_arakoon_checkup():
         StorageDriverController._voldrv_arakoon_checkup(False)
 
     @staticmethod
-    @celery.task(name='ovs.storagedriver.manual_voldrv_arakoon_checkup')
+    @ovs_task(name='ovs.storagedriver.manual_voldrv_arakoon_checkup', ensure_single_info={'mode': 'DEFAULT'})
```

Comment:
Will this prevent both `scheduled_voldrv_arakoon_checkup` and `manual_voldrv_arakoon_checkup` to be executed at the same time? I'd expect that they can now run together as their name is different.